### ISSUE TITLE
Feature | Replace the repositories card design

### ISSUE THREAD

**Issue by sambitsingha** [bishalde/GithubCrawler#8]
The repositores card design , which shows the repos of the particular github account , seems outdated. I suggest on giving it a neww design. 

Along I suggest new UI for the home page , without effecting any functionability of the current website

## I would like to work on this issue undner GSSoC'24

**Comment by nishant0708** [comment]
hey  @sambitsingha  can you assign me this issue

**Comment by sambitsingha** [comment]
@nishant0708 I am not the Project admin here. Also I have requested to work on this issue. Thank you

**Comment by AbdulhameedSk** [comment]
@bishalde I want to work on this issue. I had similar type of project this but i made it while in my learning journey  have a look 
https://git-view-kappa.vercel.app/

**Comment by SumitKumar-17** [comment]
@bishalde I want to work on this. Can you assign this issue to me under GSSoc'24?

**Comment by bishalde** [comment]
Thanyou @sambitsingha  for finding this issue.
Sure, you can proceed with it.!

Create a new branch, push the changes in that branch!
And feel free to give this Repository a STAR⭐ and follow back 😊.

**Comment by sambitsingha** [comment]
@bishalde yes I will start working on it. In the mean time , please add the GSSoC'24 and level labels
